migrations/20210610080949_suspension_reasons.js

```diff
@@ -0,0 +1,15 @@
+exports.up = function (knex) {
+  return knex.schema.createTable('suspension_reasons', (table) => {
+    table.uuid('id').primary().defaultTo(knex.raw('uuid_generate_v4()'));
+    table.uuid('psychologistId')
+        .references('dossierNumber')
+        .inTable('psychologists');
+    table.text('reason').notNullable();
+    table.date('until').notNullable();
+    table.datetime('createdAt').notNullable().defaultTo(knex.fn.now());
```

Comment:
The purpose of this table is for history right? If so, wouldn't it be hard to use if the user reactivated its account manually?

Reply:
it would... but do we really need it ? 
if yes, I suggest to add it in a further feature